app/src/main/java/com/kickstarter/libs/utils/extensions/StringExt.kt

```diff
@@ -0,0 +1,56 @@
+package com.kickstarter.libs.utils.extensions
+
+import android.util.Patterns
+import java.util.*
+
+    const val MINIMUM_PASSWORD_LENGTH = 6
+
+    /**
+     * Returns a boolean that reflects if the string is an email address
+     */
+    fun String.isEmail(): Boolean {
+        return Patterns.EMAIL_ADDRESS.matcher(this).matches()
+    }
+
+    /**
+    * Returns a boolean that reflects if the string is empty or the length is zero when space
+    * characters are trimmed
+    */
+    fun String?.isEmpty(): Boolean {
+        return this == null || this.trim().length == 0
+    }
+
+    /**
+    * Returns a boolean of if the string is not empty
+    */
+    fun String?.isPresent(): Boolean {
```

Comment:
(suggestion here) could this by an inline function?, seems is gonna be less verbose that way